bots/github/__init__.py

```diff
@@ -162,15 +162,19 @@ def request(self, method, resource, data="", headers=None):
 
     def get(self, resource):
         headers = { }
-        cached = self.cache.read(self.qualify(resource))
+        qualified = self.qualify(resource)
+        cached = self.cache.read(qualified)
         if cached:
+            if self.cache.current(qualified):
```

Comment:
Nope. Currency doesn't invalidate the cache, only what callers assume they can do with cached data. See above.